src/dotnet/Notification.Service/Db/DbChatSubscription.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Stl.Versioning;
+using Microsoft.EntityFrameworkCore;
+
+namespace ActualChat.Notification.Db;
+
+[Table("ChatSubscriptions")]
+[Index(nameof(UserId), nameof(ChatId))]
+public class DbChatSubscription : IHasId<string>, IHasVersion<long>
+{
+    public DbChatSubscription() { }
+
+    [Key] public string Id { get; set; } = null!;
+
```

Comment:
Why do you guys put empty lines between 1-line props? Just curious. 

IMO a "block" of props is actually more readable than N one-liners...

Reply:
We put empty lines between onliners and multi-line props.
Long enough attributes are placed on a separate line making the whole prop multiline